app/src/main/java/com/example/appempty/SecondFragment.kt

```diff
@@ -61,7 +62,14 @@ class SecondFragment : Fragment() {
         super.onViewCreated(view, savedInstanceState)
 
         fullname.setOnClickListener {
-            findNavController().navigate(R.id.action_SecondFragment_to_listaFragment)
+
+            if (resources.getBoolean(R.bool.isTablet)) {
```

Comment:
I think you did not test your code. This logic is in the wrong place, there should not be any functionality on this screen when tablet or landscape mode. You might want to run the app and check the ListaFragment button

Reply:
<img src="https://user-images.githubusercontent.com/22406295/93973650-677db700-fd29-11ea-840f-53949367f9db.png" height="500"">
The fragment list is containing the list of user. and the task description says
_**The User should be able to click in other user and the User Details needs to update dynamically.**_

Reply:
Update de secondFragment and put the logic in the ListaFragment